app/models/mail_chimp_client.rb

```diff
@@ -5,63 +5,49 @@ class MailChimpClient
   ADOPTER_LIST_ID = '5e50e2be93'.freeze
 
   def initialize
-    @gibbon = Gibbon::API.new
+    @gibbon = Gibbon::Request.new
     @gibbon.timeout = 30
   end
 
+  def subscribe(list_id, email, merge_vars)
+    gibbon.lists(list_id).members(hashed(email)).upsert(
+      body: {
+        email_address: email,
+        status: 'subscribed',
```

Comment:
Found it, looks like status should be set to ```pending``` here, and that will trigger the double-opt in process.  

https://pbjmarketing.com/easily-disable-mailchimp-double-opt-in-in-api-3-0/